redpanda/resources/acl/resource_acl.go

```diff
@@ -161,7 +161,7 @@ func (a *ACL) Create(ctx context.Context, request resource.CreateRequest, respon
 		return
 	}
 
-	if err := a.createACLClient(ctx, model.ClusterAPIURL.ValueString()); err != nil {
```

Comment:
what was the reason to take out the context?

Reply:
New grpc replacement for Dial does not require Context.